package.json

```diff
@@ -24,7 +31,14 @@
   "dependencies": {
     "body-parser": "^1.19.1",
     "dotenv": "^16.0.0",
-    "express": "^4.17.2"
+    "express": "^4.17.2",
+    "fs": "^0.0.1-security",
+    "path": "^0.12.7",
+    "pg": "^8.7.3",
+    "pg-hstore": "^2.3.4",
+    "process": "^0.11.10",
+    "sequelize": "^6.16.1",
+    "sequelize-cli": "^6.4.1"
```

Comment:
Sequelize-cli is a dev dependency, please move it to the dev dependencies

Reply:
Done, thank you!